docs/Dialects/Comb/RationaleComb.md

```diff
@@ -33,30 +32,6 @@ substrate that may be extended with higher level dialects mixed into it.
 TODO: Simple integer types, eventually parametrically wide integer type
 `hw.int<width>`.  Supports type aliases.  See HW rationale for more info.
 
-### Zero-bit integer width is not supported
```

Comment:
Given the change, I would make this explicitly positive.  "Zero-bit integer width is supported" with notes on how i0 is handled and how ops which break on 0 are handled.  Although not a comb issuer per-se, noting what is expected in export verilog is helpful too.  Also perhaps notes that zero in types is structurally significant.